demo.py

```diff
@@ -4,4 +4,4 @@
 pyfancy().multi("Multicolored text").output()
 pyfancy().rainbow("Rainbow text").output()
 pyfancy().parse("{red foo {bold bar} baz}").output()
-print pyfancy().red("Hello").raw(", ").blue("world!").strip()
+print(pyfancy().red("Hello").raw(", ").blue("world!").strip())
```

Comment:
What are you doing here? You ended the parentheses for `print` in the `strip` method.

Reply:
@baranskistad I think you just misunderstood the edit because of the way GitHub highlighted it.

Reply:
Yeah, sorry bout that. :P